packages/core/src/utilities/scroll/isScrollingDisabled.ts

```diff
@@ -0,0 +1,12 @@
+import {getWindow} from '@dnd-kit/utilities';
+
+import {hasOverflowStyles} from './hasOverflowStyles';
+
+export function isScrollingDisabled(
```

Comment:
I created this dedicated function because using a single `isScrollable` function containing both checks doesn't work for the following reasons:

- `isScrollingEnabled` needs to check if scrolling was explicitly enabled.
- `isScrollingDisabled` needs to check if scrolling was explicitly disabled.

Combining both into `Boolean(isScrollingEnabled && !isScrollingDisabled)` (in a single `isScrollable` function) would imply the element won't be considered scrollable unless it explicitly had scrolling enabled, which is not something that is wanted when checking `node.scrollableElement` in `getScrollableAncestors.ts` because that element should always be considered scrollable, unless scrolling is explicitly disabled.